lms/static/scripts/frontend_apps/components/EmailNotificationsPreferences.tsx

```diff
@@ -0,0 +1,96 @@
+import type { PanelProps } from '@hypothesis/frontend-shared';
+import { Button, Checkbox, Panel } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import type { StateUpdater } from 'preact/hooks';
+import { useCallback } from 'preact/hooks';
+
+import type { EmailNotificationsPreferences, WeekDay } from '../config';
+
+const dayNames: [WeekDay, string][] = [
+  [7, 'Sunday'],
+  [1, 'Monday'],
+  [2, 'Tuesday'],
+  [3, 'Wednesday'],
+  [4, 'Thursday'],
+  [5, 'Friday'],
+  [6, 'Saturday'],
```

Comment:
Do these numbers align with the values that the Python code uses on the backend to represent days of the week? I think @seanh mentioned that some standard Python function is used to define the convention. Looking at the docs I see there are a couple of functions in the stdlib - https://docs.python.org/3/library/datetime.html#datetime.date.weekday. These use different conventions. It would be good to align here to save confusion when switching between frontend and backend code.

Reply:
Also, if the values are aligned, there should be a comment referencing this - eg. "Page numbering matches `datetime.isoweekday` in Python" (or whatever the basis of the convention is).

Reply:
They do, but this has actually changed since I first implemented it due to SQLAlchemy limitations. 

Days will not be identified by indexes in the end, but by this, which is less controversial and more intuitive:

```json
{
  "instructor_email_digests.days.mon": true,
  "instructor_email_digests.days.tue": true,
  "instructor_email_digests.days.wed": true,
  "instructor_email_digests.days.thu": true,
  "instructor_email_digests.days.fri": true,
  "instructor_email_digests.days.sat": true,
  "instructor_email_digests.days.sun": true,
}
```

This PR still needs to be adapted to this change.